Src/Plugins/Security/Security.csproj

```diff
@@ -59,8 +55,17 @@
     </Reference>
   </ItemGroup>
 
-  <Target Name="CopyingSpam" AfterTargets="Build">
+  <Target Name="CopyingSecurityToSpam" AfterTargets="Build">
+
+    <ItemGroup>
+      <SecurityFile Include="$(OutputPath)Security.dll" />
+      <SpamFiles Include="$(MSBuildThisFileDirectory)*.json" />
+      <SdkFile Include="$(OutputPath)Sarif.PatternMatcher.Sdk.dll" />
+    </ItemGroup>
+
+    <Copy SourceFiles="@(SdkFile)" DestinationFolder="$(SolutionDir).spam\Security\" OverwriteReadOnlyFiles="true" SkipUnchangedFiles="false"  ContinueOnError="true" />
     <Copy SourceFiles="@(SpamFiles)" DestinationFolder="$(SolutionDir).spam\Security\" OverwriteReadOnlyFiles="true" SkipUnchangedFiles="false" />
+    <Copy SourceFiles="@(SecurityFile)" DestinationFolder="$(SolutionDir).spam\Security\" OverwriteReadOnlyFiles="true" SkipUnchangedFiles="false" ContinueOnError="true" />
```

Comment:
>ContinueOnError="true" [](start = 147, length = 22)

there are times it has a race condition, generating an error. but the dll is copied to the  dest folder. so, lets ignore as well.